### DISCUSSION TITLE
held keys in vim

### DISCUSSION THREAD

**Discussion by jonch070** [manna-harbour/miryoku#276] [Q&A] [answered]
Hello, just curious about the conflict of holding hjkl (equivalent) keys in vim and homerow mods. Its nice to hold down j and go down super fast in vim, but its a homerow mod.  How do you get around this? I imagine such a solution might also solve the issue of needing to hit modified one handed key commands. 

on an ergodox https://configure.zsa.io/ergodox-ez/layouts/Zebwv/nVQgb/0

**Comment by KAGEYAM4** [comment, marked as answer]
you can look at this, i found a similar discussion regarding vim - https://github.com/manna-harbour/miryoku/discussions/80
